Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bincode"
-version = "0.4.0"
+version = "0.4.1"
```

Comment:
Yep.  I don't know how this sneaked in.  `0.4.1` was the release that fixed the dependencies.  This PR is certainly a breaking change.